### ISSUE TITLE
Postman?

### ISSUE THREAD

**Issue by ejb** [basilesimon/using-an-api-tutorial#6]
I think the draft as it stands right now is plenty for beginners. [Do we need to even mention Postman here?](https://github.com/basilesimon/using-an-api-tutorial/blob/master/tutorial.md#tools) Could add a section at the end for those already familiar with jQuery AJAX requests, linking off to Postman and the full Juicer docs.

**Comment by basilesimon** [comment]
As is everything that's in the draft, it's open for discussion!

My intention was to include before the actual JS part a section with some sample queries from the Juicer that you can run in Postman, just to get a feel. 

It may be useless, though, and it could be worth hammering down the jQuery road straight away, with additional infos at the end.

**Comment by ejb** [comment]
Yeah, I think leaving Postman for the advanced users at the end would be for the best.
